From 24f5e5c2d899eac5daa2a4d80b840622c5d7b77d Mon Sep 17 00:00:00 2001
From: Francois-Xavier Coudert <fxcoudert@gmail.com>
Date: Fri, 27 Dec 2024 14:20:58 +0100
Subject: [PATCH] Specify dynamic version in pyproject.toml

---
 pyproject.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyproject.toml b/pyproject.toml
index a0ee318..25b3ece 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,6 @@
 [project]
 name = "libipld"
+dynamic = ["version"]
 description = "Python binding to the Rust IPLD library"
 authors = [
     {name = "Ilya (Marshal)", email = "ilya@marshal.dev"}
-- 
2.39.5 (Apple Git-154)

